### PR TITLE
Update repo README

### LINKED ISSUE #324: Update repo README
Update repo README with better format and instructions

### PR DESCRIPTION
**Pull request by ioangatop**
Closes #324

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,79 +6,147 @@
...
 
-### Run *eva*
+_eva_ can be used directly from the terminal as a CLI tool as follows:
+```sh
+eva {fit,predict,predict_fit} --config url/or/path/to/the/config.yaml 
+```
 
-Now you can run a complete *eva* workflow, for example with:
+For example, to perform a downstream evaluation of DINO ViT-S/16 on the BACH dataset with linear probing, by first infer the embeddings and perform 5 sequential fits, execute:
+```sh
```
Comment (roman807) [inline README.md:73]: --> For example, to perform a downstream evaluation of DINO ViT-S/16 on the BACH dataset with linear probing by first inferring the embeddings and performing 5 sequential fits, execute:

File: README.md [hunk 2]
```diff
@@ -6,79 +6,147 @@
...
+eva predict_fit --config https://raw.githubusercontent.com/kaiko-ai/eva/main/configs/vision/dino_vit/offline/bach.yaml
 ```
-eva fit --config configs/vision/dino_vit/online/bach.yaml 
+
+> [!NOTE] 
+> All the datasets the support automatic download in the repo have by default the option to `false` and thus you manually have to set the argument to `true`, that is `download=true`.
+
```
Comment (roman807) [inline README.md:79]: --> All the datasets that support automatic download in the repo have by default the option to automatically download set to `false`. For automatic download you have to manually set `download=true`.

**Review by roman807 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -6,79 +6,147 @@
...
+  <a href="#contributing">Contribute</a> •
+  <a href="#acknowledgements">Acknowledgements</a>
 </p>
 
 </div>
 
----
+### Introduction
+
+`eva` is an evaluation framework for oncology foundation models (FMs) by [kaiko.ai](https://kaiko.ai/). Check out the [documentation](https://kaiko-ai.github.io/eva/) for more information.
```
Comment (nkaenzig) [inline README.md:27]: `eva` vs _eva_ below

File: README.md [hunk 4]
```diff
@@ -6,79 +6,147 @@
...
-This will:
 
- - Download and extract the dataset, if it has not been downloaded before.
- - Fit a model consisting of the frozen FM-backbone and a classification head on the train split.
- - Evaluate the trained model on the validation split and report the results.
+For more information, please refer to the [documentation](https://kaiko-ai.github.io/eva/dev/user-guide/tutorials/offline_vs_online/) and [tutorials](https://kaiko-ai.github.io/eva/dev/user-guide/advanced/replicate_evaluations/).
+
+## Benchmarks
+
+In this section you will find model benchmarks which were generated with _eva_.
```
Comment (nkaenzig) [inline README.md:90]: Maybe we could provide a link to the page in documentation here which shows how to reproduce this?

File: docs/images/eva-process.gif
Comment (nkaenzig) [inline docs/images/eva-process.gif:1]: Nice graphic :) Two observations:
- Foundation*al* Model v.s. Foundation Model used in other parts of the readme / docs
- I liked the diagram that @roman807 had in the presentation, as it shows a bit more details. But I think using either one of those should be fine.

File: README.md (hunk 1 above)
Comment (ioangatop) [reply README.md:73]: nice done

File: README.md (hunk 2 above)
Comment (ioangatop) [reply README.md:79]: nice done

File: README.md (hunk 3 above)
Comment (ioangatop) [reply README.md:27]: nice done

File: README.md (hunk 4 above)
Comment (ioangatop) [reply README.md:90]: good idea, maybe we need a dedicated  tab for that 🤔

File: docs/images/eva-process.gif
Comment (ioangatop) [reply docs/images/eva-process.gif:1]: good point, a fixed! lets see for the 2nd we might indeed find smthing better

**Pull request merged**